test/gurka/gurka.h

```diff
@@ -582,18 +592,6 @@ findEdge(valhalla::baldr::GraphReader& reader,
   return std::make_tuple(baldr::GraphId{}, nullptr, baldr::GraphId{}, nullptr);
 }
 
-std::tuple<const baldr::GraphId,
-           const baldr::DirectedEdge*,
-           const baldr::GraphId,
-           const baldr::DirectedEdge*>
-findEdge(const map& map,
-         const baldr::GraphId& tile_id,
-         const std::string& way_name,
-         const std::string& end_node) {
-  baldr::GraphReader reader(map.config.get_child("mjolnir"));
```

Comment:
this was unsafe because it can return pointers to memory that is no longer around once it goes out of scope